x-pack/plugins/security_solution/public/management/pages/policy/view/components/policy_endpoint_count.tsx

```diff
@@ -59,11 +59,15 @@ export const PolicyEndpointCount = memo<
   const clickHandler = useNavigateByRouterEventHandler(toRoutePathWithBackOptions);
 
   if (nonLinkCondition) {
-    return <EuiText size="s">{children}</EuiText>;
+    return (
+      <EuiText size="s" data-test-subj={dataTestSubj}>
```

Comment:
Would be worth adding a suffix here and on the one below to know what exactly it is

Reply:
when you say suffix do you mean a doc comment or something else?

Reply:
something like: 

```jsx
 <EuiText size="s" data-test-subj={`${dataTestSubj}_WHATEVER_MORE_SPECIFIC`}>
```

but let me know if this does not make sense here